src/dma/collector/workflows/readiness_check/_postgres/main.py

```diff
@@ -136,6 +150,133 @@ def _version_check(self) -> None:
                     f"Version {self.db_version} is supported.  Please ensure that you selected a version that meets or exceeds version {detected_major_version!s}.",
                 )
 
+    def _check_pglogical_installed(self) -> None:
+        rule_code = "PGLOGICAL_INSTALLED"
+        if self.db_version is None:
+            msg = "Database version was not set."
+            raise ApplicationError(msg)
+        result = self.local_db.sql(
+            "select count(*) from collection_postgres_extensions where extension_name = 'pglogical'"
+        ).fetchone()
+        is_installed = result[0] > 0 if result is not None else False
+        for c in self.rule_config:
+            if is_installed:
+                self.save_rule_result(
+                    c.db_variant,
+                    rule_code,
+                    "ERROR",
+                    "The `pglogical` extension is not installed on the database.",
+                )
+            else:
+                self.save_rule_result(
+                    c.db_variant,
+                    rule_code,
+                    "PASS",
+                    "`pglogical` is installed on the database",
+                )
+
+    def _check_wal_level(self) -> None:
+        rule_code = "WAL_LEVEL"
+        if self.db_version is None:
+            msg = "Database version was not set."
+            raise ApplicationError(msg)
+        wal_level = self._is_rds()
```

Comment:
this is checking rds instead of wal_level

Reply:
I think I've cleaned up all of these inconsistencies if you wanted to review it again.